AR-Framework/src/at/fhv/itb6/arp/inputInterface/InputDetection.java

```diff
@@ -31,7 +34,7 @@ public InputDetection(InputConfiguration inputConfiguration){
      * @throws GamebordersNotDetectedException
      * @throws NoMarkerDetectedException
      */
-    public InputAction getUserInput() throws GamebordersNotDetectedException, NoMarkerDetectedException {
+    public InputAction getUserInput(CursorStatusListener callback) throws GamebordersNotDetectedException, NoMarkerDetectedException {
```

Comment:
We could wrap the getUserInput method into another method that resolves the click coordinates into gameboard positions